src/main/resources/application-dev.properties

```diff
@@ -2,7 +2,7 @@ spring.datasource.url=jdbc:h2:mem:testdb
 #spring.datasource.url=jdbc:h2:file:/data/demo
 spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
-spring.datasource.password=password
+spring.datasource.password=
```

Comment:
Important to note that I've changed the password to be blank for our developer database, since it's annoying as hell having to type in password every time